spec/requests/portfolio_items_spec.rb

```diff
@@ -201,38 +201,4 @@
       end
     end
   end
-
-  context "icons" do
```

Comment:
@lindgrenj6 Can we get rid of the icon_id variable from this spec, since the icons spec is now separate.